src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/ProcessedEnvelopeNotificationHandler.java

```diff
@@ -162,10 +162,17 @@ private MessageProcessingResult tryProcessMessage(IMessage message) {
 
     private ProcessedEnvelope readProcessedEnvelope(IMessage message) throws IOException {
         try {
-            return objectMapper.readValue(
-                MessageBodyRetriever.getBinaryData(message.getMessageBody()),
-                ProcessedEnvelope.class
+            ProcessedEnvelope processedEnvelope = objectMapper.readValue(
+                    MessageBodyRetriever.getBinaryData(message.getMessageBody()),
+                    ProcessedEnvelope.class
             );
+            log.info(
+                    "Parsed processed envelope message, Id :{}, Processed ccd reference :{}, Processed Ccd Type : {}",
```

Comment:
we already know it's _processed_ from first sentence. i guess `CCD reference` and `CCD type` is enough here at the end ;)

Reply:
i wanted to keep names as it is, but changing